src/components/MegaMenu/MegaMenu.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="mx-auto inline-block relative z-50 mx-0 px-0"
+    data-testId="menu-container"
+    @mouseenter="showNav = true"
+    @mouseleave="showNav = false"
+    @click="$emit('toggleMobileNavs')"
+  >
+    <div
+      id="w-dropdown-toggle-3"
+      class="outline-none mx-0 py-2.5 px-3 transition-colors duration-200 ease-linear text-gray-700 whitespace-nowrap active:text-primary-300 active:no-underline hover:text-primary-300 hover:no-underline inline-block cursor-pointer relative"
+      aria-controls="w-dropdown-list-3"
```

Comment:
Did you get a chance to research the aria stuff for this component? Did we figure out if this is needed?

Reply:
Accessibility:

- `id, aria-controls` in the toggle, and `id, aria-labelledby` in the dropdown are used for the relationship between the two. So I added a prop id to dynamically create these ids instead of using fixed ones which would cause trouble.
- updated `role` to `role: 'navigation'`
- `aria-haspopup="menu"` is useful as it "indicates that the element has a popup context menu or sub-level menu."
- The only one I'm having trouble with is `aria-expanded` (which is useful too). Currently it's fixed as false (which is not only useless, but damaging - it better be gone than fixed like that).
Anyway, as we don't have a JS variable for the current screen width, I'm not sure how to create a JS variable for the `expanded` state depending on the current screen width (to determine whether to check for `showNav` or for `showMobileNav`).